node-src/lib/getDependentStoryFiles.test.ts

```diff
@@ -2,26 +2,30 @@
 import chalk from 'chalk';
 import { afterEach, describe, expect, it, vi } from 'vitest';
 
-import * as git from '../git/git';
+import { Context } from '../types';
 import { getDependentStoryFiles, normalizePath } from './getDependentStoryFiles';
 
-vi.mock('../git/git');
-
 const CSF_GLOB = String.raw`./src sync ^\.\/(?:(?!\.)(?=.)[^/]*?\.stories\.js)$`;
 const VITE_ENTRY = '/virtual:/@storybook/builder-vite/storybook-stories.js';
 const statsPath = 'preview-stats.json';
 
 const log = { info: vi.fn(), warn: vi.fn(), error: vi.fn(), debug: vi.fn() };
-const getContext: any = ({
-  configDir,
-  staticDir,
-  ...options
-}: { configDir?: string; staticDir?: string } = {}) => ({
+const getContext: any = (
+  {
+    configDir,
+    staticDir,
+    ...options
+  }: { configDir?: string; staticDir?: string } & Context['options'] = {} as any
+) => ({
   log,
-  options: { storybookBaseDir: '.', ...options },
+  options,
   turboSnap: {},
-  storybook: { configDir, staticDir },
-  git: {},
+  storybook: {
+    baseDir: options.storybookBaseDir ? options.storybookBaseDir.replace(/^\.[/\\]?/, '') : '',
```

Comment:
Could you throw a comment on here about that regex?

Reply:
I got rid of this. I think it's simpler to just pass up the string the user has configured and do any re-mapping on the backend.